### PR TITLE
fix: watch rename syscall on Linux

### PR DESCRIPTION
**Pull request by fxha**
| Q                | A
| ---------------- | ---
| Bug fix?         | yes
| License          | MIT

### Description

Please see [this comment](https://github.com/marktext/marktext/pull/796#issuecomment-475974874). I added also `MARKTEXT_DEBUG_VERBOSE` to print debug stuff in production if we need more information for issues.

### CODE REVIEW

File: src/renderer/store/editor.js [hunk 1]
```diff
@@ -635,6 +635,8 @@ const actions = {
 
   LISTEN_FOR_FILE_CHANGE ({ commit, state, rootState }) {
     ipcRenderer.on('AGANI::update-file', (e, { type, change }) => {
+      // TODO: Set `isSaved` to false.
+      // TODO: A new "changed" notification from different files overwrite the old notification - the old notification disappears.
```
Comment (Jocs) [inline src/renderer/store/editor.js]: Yes, this is a problem!

Comment (Jocs) [reply src/renderer/store/editor.js]: It's better to notice the changes in different tab pages.

Comment (Jocs) [reply src/renderer/store/editor.js]: Let's fix this in the next 0.15.xx

File: src/main/globalSetting.js [hunk 2]
```diff
@@ -1,11 +1,8 @@
...
- * https://simulatedgreg.gitbooks.io/electron-vue/content/en/using-static-assets.html
- */
-
+// Set `__static` path to static files in production
 if (process.env.NODE_ENV !== 'development') {
   global.__static = require('path').join(__dirname, '/static').replace(/\\/g, '\\\\')
 }
 
 global.MARKTEXT_DEBUG = process.env.MARKTEXT_DEBUG || process.env.NODE_ENV !== 'production'
+global.MARKTEXT_DEBUG_VERBOSE = global.MARKTEXT_DEBUG && process.env.MARKTEXT_DEBUG_VERBOSE
```
Comment (Jocs) [inline src/main/globalSetting.js]: Why add `MARKTEXT_DEBUG_VERBOSE `if we already have `MARKTEXT_DEBUG `?

Comment (fxha) [reply src/main/globalSetting.js]: To be more verbose and prevent spam in debug mode.

**Pull request merged**